jobs/cloud_controller_ng/templates/shutdown_drain.rb.erb

```diff
@@ -6,8 +6,5 @@ $LOAD_PATH.unshift('/var/vcap/packages/cloud_controller_ng/cloud_controller_ng/l
 require 'cloud_controller/drain'
 
 @drain = VCAP::CloudController::Drain.new('/var/vcap/sys/log/cloud_controller_ng')
-@drain.log_invocation(ARGV)
```

Comment:
is there a particular reason to remove this log?

Reply:
The `bin/shutdown_drain` script is never invoked with any arguments. So the log contained useless "Drain invoked with " messages.